frescobaldi_app/app.py

```diff
@@ -228,6 +228,21 @@ def displayhook(obj):
     if obj is not None:
         print(repr(obj))
 
+
+_extensions = None
+
+def load_extensions(mainwindow):
+    """Called from MainWindow.__init__()"""
+    import extensions
+    global _extensions
```

Comment:
The use of global variables gives me a warning sign. But in this case it's following the rest of the code and could perhaps be seen as a constant object?

Reply:
Yes, that was the rationale: implement the caching like with other comparable objects.